### PR TITLE
handle displaying accounts not created by wallet in authz details

### PR DESCRIPTION
**Pull request by chriswayoub**
fixes issue #87

### CODE REVIEW

File: contexts/AuthzContext.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ export function AuthzContextProvider({children}: {children: React.ReactNode}) {
     proposer: accounts[fcl.withPrefix(voucher.proposalKey.address)],
     payer: accounts[fcl.withPrefix(voucher.payer)],
     authorizers: voucher.authorizers.map(
-      authorizer => accounts[fcl.withPrefix(authorizer)]
+      authorizer => accounts[fcl.withPrefix(authorizer)] ?? {type: 'ACCOUNT', address: fcl.withPrefix(authorizer), label: fcl.withPrefix(authorizer), keyId: 0, scopes: []}
```
Comment (psiemens) [inline contexts/AuthzContext.tsx:124]: Do we have access to the keyId (key index) here? It'd be good to avoid hardcoding it as zero.

**Review by psiemens — COMMENTED** [review]
Thanks for this improvement @chriswayoub!

File: contexts/AuthzContext.tsx [hunk 2]
```diff
@@ -121,7 +121,7 @@ export function AuthzContextProvider({children}: {children: React.ReactNode}) {
     proposer: accounts[fcl.withPrefix(voucher.proposalKey.address)],
```
Comment (kazarena) [inline contexts/AuthzContext.tsx:121]: First of all, thank you for submitting this PR! We've encountered this issue while working with multi-sig transactions where one of the accounts is not originated in Dev Wallet.

Both `proposer` and `payer` also need to be updated the same way as you did for authorizers in line 124.

**Comment by sideninja** [comment]
@justinbarry can your team check this PR I think it should either be merged or closed as it's pretty stale.

**Comment by justinbarry** [comment]
Closing as stale.

**Pull request closed without merging**